apps/frontend/src/modules/points-of-interest/points-of-interest-api.ts

```diff
@@ -36,6 +37,25 @@ class PointOfInterestApi extends BaseHTTPApi {
 
 		return (await response.json()) as APIResponse<PointsOfInterestResponseDto>;
 	}
+
+	public async findPaginated(
+		payload: PaginationQuery,
+	): Promise<APIResponse<PointsOfInterestPaginatedResponseDto>> {
+		//Didn't find implemented helper for creating search params
+		const searchParameters = new URLSearchParams(payload);
+		const queryString = searchParameters.toString();
+
+		const response = await this.load(
+			this.getFullEndpoint("/paginated", {}) + `?${queryString}`,
+			{
+				contentType: ContentType.JSON,
+				hasAuth: true,
+				method: "GET",
+			},
```

Comment:
Shouldn't it work like this?

```suggestion
			{
				contentType: ContentType.JSON,
				hasAuth: true,
				method: "GET",
				query: { query options here }
			},
```

Reply:
should , added this abbility to basic-api on the frontend